### PR TITLE
Update docker registry to twun.io's registry chart

### PR DESCRIPTION
**Pull request by Shikachuu**
Signed-off-by: Czékus Máté <mate@picloud.hu>

## Description
Docker Registry used the old `stable` chart which was outdated and the only still updated 1:1 feature comaptible one was twuni's chart.

## Motivation and Context
- [X] I have raised an issue to propose this change, which has been given a label of `design/approved` by a maintainer ([required](https://github.com/alexellis/arkade/blob/master/CONTRIBUTING.md)) #274 

## How Has This Been Tested?
Manually by running `arkade install docker-registry` with all the flags.

## Are you a GitHub Sponsor yet (Yes/No?)

- [ ] Yes
- [X] No

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Documentation

- [ ] I have updated the list of tools in README.md if (required) with `./arkade get -o markdown`
- [ ] I have updated the list of apps in README.md if (required) with `./arkade install --help`

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I've read the [CONTRIBUTION](https://github.com/alexellis/arkade/blob/master/CONTRIBUTING.md) guide
- [X] I have signed-off my commits with `git commit -s`
- [ ] I have added tests to cover my changes.
- [X] All new and existing tests passed.

- [X] I have tested this on arm, or have added code to prevent deployment

### CODE REVIEW

**Comment by canterberry** [comment]
Thanks for the tag @Shikachuu!

The twun.io domain is privately held and operated by myself, primarily for package distribution under appropriate subdomains (`helm.twun.io` among them). As you can see, there's no HTTP front-end of consequence at the parent domain.

This chart is also available via a GitHub-owned domain (see https://github.com/twuni/docker-registry.helm/issues/22#issuecomment-1004519481 for details), so if you prefer, you can use `https://twuni.github.io/docker-registry.helm`. I dual-publish each release to both repos, with provenance, and intend to continue supporting both. FWIW, of the two URLs, the GitHub-owned one is _more likely_ to transition toward community governance one day.

My primary goals for maintaining this chart are:

 * to maintain compatibility with users of the `stable` chart
 * to ensure compatibility with modern Docker+Helm+Kubernetes tooling
 * to fix any defects

So if you're using the `stable` chart, you should be fine updating to this one.

File: cmd/apps/registry_app.go [hunk 1]
```diff
@@ -92,13 +92,13 @@ func MakeInstallRegistry() *cobra.Command {
 
 		htPasswd := fmt.Sprintf("%s:%s\n", username, string(val))
 
-		err = helm.AddHelmRepo("stable", "https://charts.helm.sh/stable", updateRepo)
+		err = helm.AddHelmRepo("stable", "https://helm.twun.io", updateRepo)
```
Comment (canterberry) [inline cmd/apps/registry_app.go:95]: The repo name here should be changed as well, from `"stable"` to `"twuni"`, for consistency with the chart path on L101.

Also FYI, the URL here could be adjusted to `"https://twuni.github.io/docker-registry.helm"` if you prefer. Both will work equally, just wanted to present the option.

**Review by canterberry — APPROVED** [review]
Offering my approval, as the changeset LGTM.

**Comment by alexellis** [comment]
I spoke to @justincormack at Docker, and it seems like there is no official Helm chart for the OSS registry.

So therefore, as discussed on Discord @Shikachuu, please:

* Make it very clear in the command help messages that this is a community chart, not "official"
* Use the GitHub publishing address instead
* Link to @canterberry's GitHub repo in the post-installation notes and in the help message

Once you have that arranged, I'll merge this PR.

@canterberry out of interest, is there still support for ARM architectures?

**Comment by canterberry** [comment]
I appreciate the diligence, @alexellis ! All correct, to my knowledge.

As for ARM support: I think that depends exclusively on the image and whether it's built for ARM architecture. The chart supports custom images, so even if there wasn't an official ARM-compatible image, one could in theory compile (or locate) one and reference that instead.

I haven't seen any ARM-related changes come through in the chart, so should be status quo. Feel free to create an issue in twuni/docker-registry.helm if there's a compatibility issue we might be able to resolve in the chart.

**Comment by Shikachuu** [comment]
> I spoke to @justincormack at Docker, and it seems like there is no official Helm chart for the OSS registry.
> 
> So therefore, as discussed on Discord @Shikachuu, please:
> 
>     * Make it very clear in the command help messages that this is a community chart, not "official"
> 
>     * Use the GitHub publishing address instead
> 
>     * Link to @canterberry's GitHub repo in the post-installation notes and in the help message
> 
> 
> Once you have that arranged, I'll merge this PR.
> 
> @canterberry out of interest, is there still support for ARM architectures?

Done :heavy_check_mark:

File: cmd/apps/registry_app.go [hunk 2]
```diff
@@ -24,8 +24,8 @@ import (
 func MakeInstallRegistry() *cobra.Command {
 	var registry = &cobra.Command{
 		Use:          "docker-registry",
-		Short:        "Install a Docker registry",
-		Long:         `Install a Docker registry`,
+		Short:        "Install a community maintained Docker registry chart",
+		Long:         `Install a community maintained Docker registry chart`,
```
Comment (alexellis) [inline cmd/apps/registry_app.go:28]: I'd like to see the URL in the long description - wrapped as best as possible to 80 chars

Comment (alexellis) [reply cmd/apps/registry_app.go:28]: Link also to the Open Source Docker registry - https://github.com/distribution/distribution

Comment (alexellis) [reply cmd/apps/registry_app.go:28]: @Shikachuu PTAL when you can

Comment (Shikachuu) [reply cmd/apps/registry_app.go]: Done :heavy_check_mark:

**Review by alexellis — APPROVED** [review]
LGTM

**Comment by alexellis** [comment]
Thank you both. @canterberry your input is very much appreciated

If you're interested in linking back to `arkade install docker-registry` from your docs / README, it's a very quick and convenient way to do an automated / 1-line installation with auth/tls.

https://blog.alexellis.io/get-a-tls-enabled-docker-registry-in-5-minutes/

**Pull request merged**